packages/next/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -410,7 +415,8 @@ function getExtractMetadata(params: {
 
       const entryMetadata: EntryMetadata = {
         env: new Set<string>(),
-        wasmBindings: new Set<WasmBinding>(),
+        wasmBindings: new Set(),
+        blobBindings: new Set(),
       }
 
       for (const entryModule of entryModules) {
```

Comment:
I would rename it to `for (const module of queue)` as it's not only going through entry modules, but through all referenced modules.

Reply:
makes total sense, but can we do that in a different PR, to minimize the amount of changes per PR? :pray: